demo/full-featured.md

```diff
@@ -19,4 +21,18 @@ The following plugins are excluded from this example:
 * [Full Page]({{site.baseurl}}/plugins/fullpage) — Used for modifying HTML `<head>` elements.
 * [Paste]({{site.baseurl}}/plugins/paste) — [**PowerPaste**]({{site.baseurl}}/plugins/powerpaste) included instead.
 * [Spellchecker]({{site.baseurl}}/plugins/spellchecker) — [**Spell Checker Pro**]({{site.baseurl}}/plugins/tinymcespellchecker) included instead.
+* [Tab Focus]({{site.baseurl}}/plugins/tabfocus) — Changes the behavior of the TAB-key within the editor.
+
+## Full Featured: Non-Premium Plugins
+
+This example includes only non-premium plugins. These plugins are also used in the [Full Featured: Including Premium Plugins Demo above](#fullfeaturedincludingpremiumplugins).
+
+{% include codepen.html id="open-source-plugins" %}
+
+The following plugins are excluded from this example:
+* [All premium plugins](https://www.tiny.cloud/pricing/#demo-enterprise).
+* [Autoresize]({{site.baseurl}}/plugins/autoresize) — Resizes the editor to fit the content.
+* [BBCode]({{site.baseurl}}/plugins/bbcode) — Changes the markup used for the content.
+* [Full Page]({{site.baseurl}}/plugins/fullpage) — Used for modifying HTML `<head>` elements.
+* [Spellchecker]({{site.baseurl}}/plugins/spellchecker) — Requires [Server-Side components]({{site.baseurl}}/enterprise/server/).
```

Comment:
This probably needs to link to https://www.tiny.cloud/docs/general-configuration-guide/spell-checking/, as that link is for the premium components.

Reply:
updated